### DISCUSSION TITLE
[x][closed][manuscript][fairtracks] experiment:target

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#95] [Team Posts]
The description is:
> Main target of experiment, e.g. antibody/disease/gene/... [Needs more details]

However the spec requires a term from SO. Alas SO does not contain neither disease, antibodies...

FYI: IHEC are currently debating how to represent targets here: https://github.com/IHEC/ihec-ecosystems/issues/27

**Comment by sveinugu** [comment]
More or less duplicate of https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/15

**Comment by sveinugu** [comment]
SO is obviously not a full solution. It does, however, contain the histone modifications (with a bit strange primary forms, as mentioned below), so good enough for the example data. Also suggested in the IHEC discussion you refer to.
I suppose we should build upon the IHEC thoughts. Seems some sort of mapping from tech_type (or whatever we will call it) to target vocabulary is a possibility (basically what you suggest in IHEC context):
  * 'ChIP-seq (TF)' -> HGNC
  * 'ChIP-seq (histone)' -> controlled vocab/SO
  * 'Genome annotation' -> SO
  * 'GWAS' -> Some disease/trait ontology
...

**Comment by sveinugu** [comment]
We should try to develop a minimal solution that fits with the BLUEPRINT data and then go from there. Do you have an overview of know how many different "tech_types" are present there?

**Comment by dzerbino** [comment]
Sorry missed this thread. Blueprint has the following tech types:
- "ChIP-Seq": "http://purl.obolibrary.org/obo/OBI_0000716",
- "ATAC-seq": "http://purl.obolibrary.org/obo/OBI_0002039",
- "Bisulfite-Seq": "http://purl.obolibrary.org/obo/OBI_0000748",
- "DNase-Hypersensitivity": "http://purl.obolibrary.org/obo/OBI_0001853",
- "miRNA-Seq": "http://purl.obolibrary.org/obo/OBI_0001922",
- "RNA-Seq": "http://purl.obolibrary.org/obo/OBI_0001271",
- "WGS": "http://purl.obolibrary.org/obo/OBI_0001863"

Only ChIP-Seq really needs a target.

**Comment by sveinugu** [comment]
So I'm not into the details of all these technologies. I suppose one could at least come up with a single term describing a target which corresponds to the technique, e.g. "DNase-Hypersensitivity" will always have the target "DNAse I". For RNA-seq, for instance, I suppose there might be different RNA types targeted? 

I would argue for always having something in the target field, as this would simplify the use in later analyses. One could for instance combine tracks from different techniques into a single analysis. Having the possibility of just using the target field as labels (e.g. in a heatmap), one could then very easily get combinations like e.g. H3K4me3, H3K27me9, DNAse I, ... If the statistics has been properly done (which is our main field of expertice), combining heterogeneous tracks in this way should be little problem.

**Comment by dzerbino** [comment]
OK, I think I can fudge something.

**Comment by sveinugu** [comment]
Issue for this: https://github.com/fairtracks/fairtracks_standard/issues/11. Closing. Keeping pin for manuscript discussion
